### PR TITLE
Adds a FAQ section with an image on the left hand side

### PR DESCRIPTION
**Pull request by lizzietm**
Adds a FAQ section with an image on the left hand side (works for website, not sure if image shows above FAQs on mobile so need someone to check.)

### CODE REVIEW

File: conversationally/frontend/src/pages/landingPage/LandingPage.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import Fold from "./sections/Fold";
 import bkg from "../../../public/assistants.png";
 import { useAuth0 } from "@auth0/auth0-react";
 import CTAButton from "../../components/CTAButton";
+import FAQs from "./sections/FAQs.jsx"
```
Comment (life-efficient) [inline conversationally/frontend/src/pages/landingPage/LandingPage.jsx:7]: No need for file extensions when it's a JSX import (usually a component)

File: conversationally/frontend/src/pages/landingPage/LandingPage.jsx [hunk 2]
```diff
@@ -33,11 +34,20 @@ const LandingPage = () => {
...
+        { question: "How Many assistants can I have?", answer: "As many as you want" },
+        { question: "Are the assitants sentient?", answer: "Not yet, but watch this space" },
+    ];
+
     return (
         <>
             <Navbar />
-            <Fold 
-                headline="Create custom voice assistants you can talk with" 
+            <FAQs img='conversationally/frontend/public/wav.png' faqs={faqData} />
```
Comment (life-efficient) [inline conversationally/frontend/src/pages/landingPage/LandingPage.jsx:48]: The "fold" is defined as the part of the website you see when you first land on it, so naturally it has to come first (after the navbar)

File: conversationally/frontend/src/pages/landingPage/sections/FAQs.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+        {props.faqs.map((faq, index) => ( // Map over the faqs array to render an Accordion for each
+          <Accordion key={index} >
+
+            <AccordionSummary
+              expandIcon={<AddIcon />} // Changed to a plus icon
+              aria-controls={`panel${index}a-content`}
+              id={`panel${index}a-header`}
+            >
+              <Typography>{faq.question}</Typography>
```
Comment (life-efficient) [inline conversationally/frontend/src/pages/landingPage/sections/FAQs.jsx:54]: Always supply a variant prop to the typography component

**Review by life-efficient — CHANGES_REQUESTED** [review]

**Comment by life-efficient** [comment]
- [ ] Also, please generate a better image that could realistically be used here. The one you've got is not good enough.

**Comment by life-efficient** [comment]
Closes #78

File: conversationally/frontend/src/pages/landingPage/LandingPage.jsx (hunk 2 above)
Comment (lizzietm) [reply conversationally/frontend/src/pages/landingPage/LandingPage.jsx:48]: "fold" replaced back into it's original position.

File: conversationally/frontend/src/pages/landingPage/LandingPage.jsx (hunk 1 above)
Comment (lizzietm) [reply conversationally/frontend/src/pages/landingPage/LandingPage.jsx:7]: Extension removed.

File: conversationally/frontend/src/pages/landingPage/sections/FAQs.jsx (hunk 3 above)
Comment (lizzietm) [reply conversationally/frontend/src/pages/landingPage/sections/FAQs.jsx]: Added a h7 variant prop.

**Review by lizzietm — COMMENTED** [review]
FAQ section still has scaling issue for mobile and web but requesting to merge to find all the issues in the MVP

**Pull request merged**